source/pkeyread.c

```diff
@@ -10,7 +10,13 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
+#ifndef _WIN32
 #include <unistd.h>
+#else
+#include <windows.h>
+#include "perflib/getopt.h"
+#include "perflib/basename.h"
+#endif	/* _WIN32 */
```

Comment:
Could you please do the usual indentation here and in the other source files where you're adding this?